### PR TITLE
cert-manager-csi-spiffe-doc update to include instructions to install on openshift

### PR DESCRIPTION
**Pull request by worldhopper**
1. Updated cert-manager CSI-SPIFFE-Driver installation to include instructions to install on OpenShift
2. Updated observability (Metrics, Logging, Dashboards) and security (mTLS, JWT/Auth0-Okta, Waap/WAF and OPA policy) documentation

### CODE REVIEW

File: docs/security/security-jwt.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+_NOTE: This page provides documentation for the JWT and Client Identity validation which is a pull request pending approval and merging into QuicSec. Integration via OIDC platforms like Auth0/Okta are enabled with this capability, enabling transparent user authentication workflows on behalf of the app.
+
+## Overview: JWT Assignment (via OIDC) and Authorization (Claims Validation)
+
+User Identity workflows (using JWT) can be enabled in a single step on behalf of the applications. The JWT assignment (via OIDC) and claims authorization workflows are implemented as a plugin allowing for dynamic addition when enabled. The notable capabilities implemented by the plugin include:
+
+* OIDC Redirection: Requests missing an Authorization header are redirected via OIDC to a configured authorization service provider (e.g., Auth0/Okta).
+
+* Claims Authorization: Individual transactions will be allowed or denied based on policies matching on the audience and scope embedded in the token
```
Comment (rscampos) [inline docs/security/security-jwt.md:11]: dot in the final

File: docs/security/security-jwt.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+The types of client accesses to consider when building policies:
+
+* 1. Configure Authorization Provider (e.g., Auth0 / Okta) for this application, providing a redirection url during configuration and receiving a client secret.
+
+* 2. Clients that haven't previously authenticated with the authorization service to receive a jwt. In this case, when the client application attempts to access the App, it will be redirected to the Authorization service via OIDC. Once the client has successfully authenticated with the Auth service, it will be assigned a jwt with embedded issuer, claims and scope, and redirected to the redirection uri via OIDC in order to access the resource previously requested.
+
+* 3. Clients that have previously authenticated with the authorization service and initiate the requet with an authorization header containing a jwt. The issuer signature within the jwt is verified and the claims and scope extracted and authorized for the resource being accessed. This authorization can either be done with built-in policies, or via external authorization plugins (e.g., Open Policy Agent/OPA) using policy schema defined for the plugin (e.g., OPA rego).
```
Comment (rscampos) [inline docs/security/security-jwt.md:32]: replace "jwt" to "JWT"

File: docs/security/security-jwt.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Audit
+
+The built-in observability features provide detailed identity-enriched logs, telemetry and dashboards, as described in the [Observability](/docs/observability-dashboards) docs.
```
Comment (rscampos) [inline docs/security/security-jwt.md:36]: is "/docs/observability/observability-dashboards.md" ?

File: docs/security/security-mtls.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+
+The important element to enable mTLS is the identity of the workload, and it is enabled with the help of pluggable solutions like SPIFFE and cert-manager. Identity managemement (including cert provisioning/attestation, hourly rotation and validation) is completely automated. For mTLS, peer certificates are automatically validated as governed by policy.
+
+
+## Architecture
+
+The key ingredients for zero-trust security with mTLS are Identity, AuthN/Z, Encryption and Audit. These are all enabled in a single step, which can be further automated for zero-config security which follows the workload instantly across any platform, and independent of infrastructure. Enforcement is consistent across all appEdge bindings, including native/proxyless HTTP apps.
+
+* Identity: enabled with the help of pluggable solutions like SPIFFE and cert-manager. Identity managemement (including cert provisioning/attestation, hourly rotation and validation) is completely automated.
```
Comment (rscampos) [inline docs/security/security-mtls.md:14]: suggestion: Put Identity/AuthN/Z... in bold

File: docs/observability/observability-monitoring.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+# Observability Addon Deployment
+
+Deploying the manifests below pulls down the standard upstream versions of Prometheus, Loki and Grafana from their upstream repositories.
+
+```
+kubectl apply -f --server-side kubernetes-manifests/metrics/operator
+kubectl apply -f kubernetes-manifests/metrics/
+```
+
+Grafana, Prometheus and Loki can now be accessed via port-forward. Alternatively, an ingressgateway can be deployed for access (by **kubectl apply -f contour/60-ingress-grafana-prometheus-loki.yaml**)
```
Comment (rscampos) [inline docs/observability/observability-monitoring.md:15]: dot in the end

**Review by rscampos — COMMENTED** [review]
just some suggestions

File: docs/security/security-mtls.md (hunk 4 above)
Comment (worldhopper) [reply docs/security/security-mtls.md:14]: Is bold alread?

File: docs/observability/observability-monitoring.md (hunk 5 above)
Comment (worldhopper) [reply docs/observability/observability-monitoring.md:15]: Not a biggie.

**Review by rscampos — APPROVED** [review]

**Pull request merged**